src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
```

Comment:
`.js-` prefixed classes should be used for js hooks only. When styling them, just use the module name.